alfalfa_web/server/resolvers.js

```diff
@@ -243,8 +243,8 @@ async function sitesResolver(user, siteRef, context) {
           datetime: ""
         };
         const site_hash = await getHash(context.redis, site.siteRef);
-        if (site_hash && site_hash["sim_time"]) {
```

Comment:
this was an important check, cause sometimes the site_hash is undefined. Without this check the sites view breaks